packages/react-native/local-cli/generator-macos/templates/macos/HelloWorld-macOS/AppDelegate.mm

```diff
@@ -1,6 +1,7 @@
 #import "AppDelegate.h"
 
 #import <React/RCTBundleURLProvider.h>
+#import <ReactAppDependencyProvider/RCTAppDependencyProvider.h>
```

Comment:
If you're not using codegen, will this exist?

You can use the diff I provided in #2558 if you want to be careful.

Reply:
Good catch.. I guess CI will tell me if paper fails :D